website/docs/metrics.md

```diff
@@ -51,6 +51,14 @@ Below are the list of metrics provided by Gatekeeper:
 
     Aggregation: `Distribution`
 
+## Expansion Template
+
+- Name: `gatekeeper_expansion_templates`
```

Comment:
Do we plan to add status later? if so, perhaps open an issue for it?

Reply:
Yep, good call. I think at the minimum, we should add the status subresource to include a `byPod` status, which would show if the `ExpansionTemplate` is enforced by each Pod. 

I opened #2302 to track this.